Makefile

```
@@ -7,9 +7,7 @@ export $(shell sed 's/=.*//' ./.env)
 export NODE_ENV ?= development
 
 
-.PHONY: build-common watch-common app-local print-env run-local start-local-services
-
-run-local: | build-common app-local 
+.PHONY: build-common watch-common app-local print-env start-local-services
```

Still not sure if we need to build-common before watch-common

---

Can we make these more consistent?
```suggestion
.PHONY: print-env watch-common start-local-apps start-local-services
```